index.html

```diff
@@ -70,6 +71,179 @@ <h1>MUHYIDEEN ELIAS</h1>
       <hr>
     </div>
   </section>
+  <section class="section-3">
+    <img src="./images/portfolio-bg.svg" alt="portfolio-bg">
+    <div class="card">
+      <div class="card-img">
+        <div class="github"><img src="./images/github.svg" alt="github"></div>
+        <div class="linkedin"><img src="./images/linkedin.svg" alt="linkedin"></div>
+        <div class="two-fingers"><img src="./images/two-fingers.svg" alt="two-fingers"></div>
+        <div class="twitter"><img src="./images/twitter.svg" alt="twitter"></div>
+        <div class="medium"><img src="./images/medium.svg" alt="medium"></div>
+      </div>
+      <div class="arrow-card">
+        <div class="arrow"><img src="./images/arrow.svg" alt="work"></div>
+        <span>Work</span>
+      </div>
+    </div>
+    <div class="my-recent-work">
+      <h4>MY RECENT WORK</h4>
+    </div>
+    <div class="breakage">
+      <hr>
+    </div>
+  </section>
+  <section class="section-4">
+    <div class="project">
+      <div class="featured-project">
+        <div class="recipe-card">
+          <img src="./images/project-img/drop.svg" alt="recipe">
+        </div>
+        <div class="details-card">
+          <div class="project-header">
+            <h4>Natural Drop</h4>
+          </div>
+          <div class="project-body">
+            <span>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint laboriosam minus sunt expedita voluptatum
+              est repudiandae nesciunt soluta consectetur, iure quia, provident nostrum optio atque quas itaque ut
+              reprehenderit omnis?
+            </span>
+            <div class="programming-languages">
+              <div class="css">css</div>
+              <div class="html">html</div>
+              <div class="bootrap">boostrap</div>
+              <div class="ruby">Ruby</div>
+            </div>
```

Comment:
- [ ] I suggest you consider using `<ul> <li>` for this part of the project, reason being that this type of list presentation improves the readability of your HTML code